web/src/androidTest/kotlin/com/google/accompanist/web/WebTest.kt

```diff
@@ -465,19 +467,18 @@ class WebTest {
 
         rule.waitForIdle()
 
-        onWebView()
-            .withElement(findElement(Locator.ID, "link"))
-            .perform(webClick())
+        // HACK: EspressoWeb webClick doesn't track back navigation for some reason
+        // so manually click the view
+        rule.onNodeWithTag(WebViewTag).performClick()
```

Comment:
How about making an extension method for this and having the comment visible once there?